src/main/kotlin/no/nav/familie/klage/vurdering/domain/Vurdering.kt

```diff
@@ -53,7 +53,26 @@ enum class Hjemmel(val kabalHjemmel: KabalHjemmel) {
     BT_FIRE(KabalHjemmel.BTRL_4),
     BT_FEM(KabalHjemmel.BTRL_5),
     BT_NI(KabalHjemmel.BTRL_9),
+    BT_TI(KabalHjemmel.BTRL_10),
+    BT_ELLEVE(KabalHjemmel.BTRL_11),
+    BT_TOLV(KabalHjemmel.BTRL_12),
     BT_TRETTEN(KabalHjemmel.BTRL_13),
-    FT_EØS(KabalHjemmel.EOES_AVTALEN),
-    FT_EØS_FOR(KabalHjemmel.EOES_883_2004_6)
+    BT_SYTTEN(KabalHjemmel.BTRL_17),
+    BT_ATTEN(KabalHjemmel.BTRL_18),
+    KS_TO(KabalHjemmel.KONTSL_2),
+    KS_TRE(KabalHjemmel.KONTSL_3),
+    KS_SEKS(KabalHjemmel.KONTSL_6),
+    KS_SYV(KabalHjemmel.KONTSL_7),
+    KS_ÅTTE(KabalHjemmel.KONTSL_8),
+    KS_NI(KabalHjemmel.KONTSL_9),
+    KS_TI(KabalHjemmel.KONTSL_10),
+    KS_ELLEVE(KabalHjemmel.KONTSL_11),
+    KS_TOLV(KabalHjemmel.KONTSL_12),
+    KS_TRETTEN(KabalHjemmel.KONTSL_13),
+    KS_SEKSTEN(KabalHjemmel.KONTSL_16),
+    UTLAND_EØS(KabalHjemmel.EOES_AVTALEN),
+    UTLAND_NORDISK(KabalHjemmel.NORDISK_KONVENSJON),
+    UTLAND_TRYGDEAVTALER(KabalHjemmel.ANDRE_TRYGDEAVTALER),
+    UTLAND_EØS_FORORDNINGEN_FEM(KabalHjemmel.EOES_883_2004_5),
+    UTLAND_EØS_FORORDNINGEN_SEKS(KabalHjemmel.EOES_883_2004_6)
```

Comment:
Endret enumnavnet for FT_EØS og FT_EØS_FOR - gitt praten vi akkurat hadde går jeg ut fra at det ikke er et problem @eirarset @ViktorGSolberg 
Disse er vel ikke tatt i bruk ennå?